vdev/src/commands/package/msi.rs

```diff
@@ -0,0 +1,58 @@
+use anyhow::Result;
+
+#[cfg(windows)]
+use {
+    crate::{app, util},
+    std::env,
+    std::fs,
+    std::iter::once,
+    std::path::Path,
+};
+
+/// Create a .msi package for Windows
+#[derive(clap::Args, Debug)]
+#[command()]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(self) -> Result<()> {
+        #[cfg(windows)]
+        {
+            // TODO: wait for other PR to be merged then replace the below line with app::version()
+            let archive_version = env::var("VERSION").or_else(|_| util::read_version())?;
```

Comment:
I believe this is merged now?

Reply:
Not yet, that change is found in: 
* https://github.com/vectordotdev/vector/pull/16724